tests/dbutils.py

```diff
@@ -4,7 +4,7 @@
 
 
 # TODO: should this be somehow be divined from environment?
-POSTGRES_USER, POSTGRES_HOST = 'postgres', 'localhost'
+POSTGRES_USER, POSTGRES_HOST = 'lele', 'localhost'
```

Comment:
Looks like you forgot this one!

Reply:
Ouch, sorry... Plz tell me what you'd prefer me doing, rebase the changes omitting this one and repush?